code/modules/bluespace_exploration/bluespace_drive.dm

```diff
@@ -0,0 +1,35 @@
+GLOBAL_LIST_INIT(bluespace_drives, list())
+GLOBAL_VAR(main_bluespace_drive)
```

Comment:
I wonder if you should just store this on the subsystem.

Reply:
Yes, definately